From f282b4a6bb0933678f1acba9cf3aa258345d44f9 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 24 Feb 2016 16:33:04 +0000
Subject: [PATCH 0659/1000] MANTIS 40061 Extend Conveyor-Component for extended
 ConveyorDev.idl

codes.idl
V1.3.7

moved BAGGAGE_INCORRECTLY_PLACED, BAGGAGE_TUB_REQUIRED, BAGGAGE_HANDLE_EXTENDED, BAGGAGE_TOO_WIDE, BAGGAGE_TOO_SMALL from 151,... to 170,...



git-svn-id: svn://localhost/SelfServiceCommon/trunk@992 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Cuss/codes.idl | 15 ++++++++++-----
 1 file changed, 10 insertions(+), 5 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/codes.idl b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
index 50a2565f..1769e60a 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/codes.idl
+++ b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
@@ -51,6 +51,11 @@ Changed:   Added baggage handling status codes (Nicolas Bruns, 29.06.2004)
 @li V1.3.6
 @li Mantis 0040061: Extend Conveyor-Component for extended ConveyorDev.idl
 @li added BAGGAGE_TOO_SMALL
+
+@date 24.02.2016
+@li V1.3.7
+@li Mantis 0040061: Extend Conveyor-Component for extended ConveyorDev.idl
+@li moved BAGGAGE_INCORRECTLY_PLACED, BAGGAGE_TUB_REQUIRED, BAGGAGE_HANDLE_EXTENDED, BAGGAGE_TOO_WIDE, BAGGAGE_TOO_SMALL from 151,... to 170,...
                
 */
 
@@ -313,11 +318,11 @@ module statuscodes
     const long BAGGAGE_DELIVERED             = 148; /**< <b>Scope:</b> private + platform  */
     const long BAGGAGE_INTERFERENCE_USER     = 149; /**< <b>Scope:</b> private + platform  */
     const long BAGGAGE_INTRUSION_SAFETY      = 150; /**< <b>Scope:</b> private + platform  */
-    const long BAGGAGE_INCORRECTLY_PLACED    = 151; /**< <b>Scope:</b> private + platform; materna ccm extension */
-    const long BAGGAGE_TUB_REQUIRED          = 152; /**< <b>Scope:</b> private + platform; materna ccm extension */
-    const long BAGGAGE_HANDLE_EXTENDED       = 153; /**< <b>Scope:</b> private + platform; materna ccm extension */
-    const long BAGGAGE_TOO_WIDE              = 154; /**< <b>Scope:</b> private + platform; materna ccm extension */
-    const long BAGGAGE_TOO_SMALL             = 155; /**< <b>Scope:</b> private + platform; materna ccm extension */
+    const long BAGGAGE_INCORRECTLY_PLACED    = 170; /**< <b>Scope:</b> private + platform; materna ccm extension */
+    const long BAGGAGE_TUB_REQUIRED          = 171; /**< <b>Scope:</b> private + platform; materna ccm extension */
+    const long BAGGAGE_HANDLE_EXTENDED       = 172; /**< <b>Scope:</b> private + platform; materna ccm extension */
+    const long BAGGAGE_TOO_WIDE              = 173; /**< <b>Scope:</b> private + platform; materna ccm extension */
+    const long BAGGAGE_TOO_SMALL             = 174; /**< <b>Scope:</b> private + platform; materna ccm extension */
 
     const long FORMAT_ERROR                = 201; /**< <b>Scope:</b> private + platform (platform only for output classes) */
     const long LENGTH_ERROR                = 202; /**< <b>Scope:</b> private + platform (platform only for output classes) */
-- 
2.41.0.windows.1

